web/app/themes/mitlib-parent/css/scss/partials/_fonts.scss

```diff
@@ -5,135 +5,135 @@ This CSS resource incorporates links to font software which is the valuable copy
 @font-face{
 	font-family:"NHaasGroteskDisp";
 	font-weight: 200;
-	src:url("Fonts/825400/45070789-5c37-49df-beb0-ab1e3133920e.eot?#iefix");
-	src:url("Fonts/825400/45070789-5c37-49df-beb0-ab1e3133920e.eot?#iefix") format("eot"),url("Fonts/825400/ca6a839e-112e-49a1-b98d-ff71960f48f8.woff2") format("woff2"),url("Fonts/825400/9683323b-2589-4b72-9201-151e5727d797.woff") format("woff"),url("Fonts/825400/c10a1505-b3d6-46b1-ba11-65d5d5c72150.ttf") format("truetype");
+	src:url("../../fonts/825400/45070789-5c37-49df-beb0-ab1e3133920e.eot?#iefix");
```

Comment:
Is there a fonts equivalent to `#{$imagesPath}` and `#{$libsPath}` that would allow us to normalize how these types of assets are referenced in the scss files? And if not could we move this font into `#{$libsPath}` like `#{$libsPath}/NHaasGroteskDisp/...`?